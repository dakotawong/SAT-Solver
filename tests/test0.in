a123

a+c
a*---a
a+b;a:0,b:1
 ( (-a)+(a*b)) * a * (c + -b) *-c
(a+b+c)*(a+b+-c)*(-b+a +c)*(-a*-c)
(a+b+c)*(a+b+-c)*(-b+a +c)*(a+-b+-c)*(-a+b+c)*(-a+b+-c)*(-a+-b+c)*(-a+-c+-c)
(a+b+c)*(a+b+-c)*(-b+a +c)*(a+-b+-c)*(-a+b+c)*(-a+b+-c)*(-a+-b+c)*(-a+-c+-b)
(a+b+c)*(a+b+-c)*(-b+a +c)*(a+-b+-c)*(-a+b+c)*(-a+b+-c)*(-a+-b+c)
VAr1 * -VAr1
VAr1 + -VAr1
 -((A*  B)+ C)
 (A+-B+C) * (B+C) * (-A+ C) * (B +-C) * -               (C)
(a1)*(-a1+a2) * (-a2+a3) *(-a3)
  (-B*-C * D) + (-B * -  D) + (C *D) + (B)         
(B+C+-D)  *(D+B)*(-C+-D)* (-B)
a*---a !
(+)
a*(b + c
a+b)
k(a+b)
a++
(-a+b+c) * (-b) * (-c)
(-a+b+c) * (-b) * (-c) * a 
-(-P+Q)*P*Q
a*--a
var+-var
a*-A
(a*b*c)+(a+c+a)*(-a+d)+(a)
 (a*b*c)+(a*c*a)+(-a*d)
   ( a  * b* -a)+(a   *c*a)+ (-a*d)
(a * b *-a) +(-a*c *a) +(- a*d )
( a*b *- a)+( -a*c*a )+( -a* d)*a
