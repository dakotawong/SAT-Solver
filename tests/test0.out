sat
Error: invalid input
sat
unsat
Error: invalid character/symbol
unsat
unsat
unsat
unsat
sat
unsat
sat
sat
unsat
unsat
sat
unsat
Error: an invalid character/symbol was found in the input
Error: invalid input
Error: missing a closing ')'
Error: invalid input
Error: invalid input
Error: invalid input
sat
unsat
unsat
sat
sat
sat
sat
sat
sat
sat
unsat
